### PR TITLE
Wifi 1127226

### PR DESCRIPTION
**Pull request by tefn3849**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=15abbde1106c)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8559d7cb7c3b)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a489552814ce)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cc727fa69fe5)

File: apps/system/test/unit/wifi_test.js [hunk 1]
```diff
@@ -538,8 +507,8 @@ suite('WiFi > ', function() {
       Wifi.wifiEnabled = false;
       navigator.battery = { charging: false };
 
-      var isWifiPowerSavingMode = false;
-      var isSetPowerSavingModeCalled = false;
+      var isWifiPowerSavingMode = true;
+      var isSetPowerSavingModeCalled = true;
```
Comment (Seinlin) [inline apps/system/test/unit/wifi_test.js]: I think initialize as true and check if the value is true later is not a good idear. Need to consider if the test case should be kept or modified properly.

File: apps/system/test/unit/wifi_test.js [hunk 2]
```diff
@@ -638,8 +607,8 @@ suite('WiFi > ', function() {
...
```
Comment (Seinlin) [inline apps/system/test/unit/wifi_test.js]: Same as above comment.

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=850b4c4f48df)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=40d6ae1c7c88)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8d87e153a70b)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3af1b84e4103)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a18495980930)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7f723b26c781)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8ea6151de636)

File: apps/system/test/unit/wifi_test.js [hunk 3]
```diff
@@ -572,20 +502,9 @@ suite('WiFi > ', function() {
       Wifi.wifiEnabled = false;
       navigator.battery = { charging: false };
 
-      var isWifiPowerSavingMode = false;
-      var isSetPowerSavingModeCalled = false;
-      var fakeDOMRequest = {
-        onsuccess: function() {},
-        onerror: function() {}
-      };
       var stubMozWifiManager = this.sinon.stub(navigator,
```
Comment (alivedise) [inline apps/system/test/unit/wifi_test.js]: nit: Looks like this stub is redundant right now. Remove it.

File: apps/system/test/unit/wifi_test.js [hunk 4]
```diff
@@ -625,27 +542,14 @@ suite('WiFi > ', function() {
...
```
Comment (alivedise) [inline apps/system/test/unit/wifi_test.js]: Again, remove this redundant stub totally and
https://github.com/tefn3849/gaia/blob/wifi_1127226/apps/system/test/unit/wifi_test.js#L323-L324

File: apps/system/test/unit/wifi_test.js [hunk 5]
```diff
@@ -672,20 +576,9 @@ suite('WiFi > ', function() {
...
```
Comment (alivedise) [inline apps/system/test/unit/wifi_test.js]: Again, remove this stub if it's redundant and the restore

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[tefn3849 Amy Chung (tefn3849)](https://github.com/tefn3849) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=32ef00f01a65)

**Pull request merged**